seokhwan/3/문자열_다루기_기본.js

```diff
@@ -0,0 +1,11 @@
+function solution(s) {
+  const arr = s.split("");
+  let res = true;
+
+  for (let i = 0; i < arr.length; i++) {
+    if (isNaN(arr[i])) {
```

Comment:
```suggestion
    if (Number.isNaN(arr[i])) {
```
isNaN() 보다 더 엄격하게 검사할 수 있는 Number.isNaN()이 있다고 합니다~~
isNaN() 같은 경우에는 몇몇 혼란스러운 상황을 발생시킬 수 있음으로
Number.isNaN()을 사용해보심도 좋을 거 같습니다~~